### PR TITLE
Adding galaxy_ng on example configs

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

**Comment by mikedep333** [comment]
Just to confirm, we do not need any other configs?

**Review by mikedep333 — APPROVED** [review]
Just to confirm, we do not need any ansible-pulp vars, or any ansible tasks for these plugins?

File: example.dev-config.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ pulp_default_admin_password: password
 pulp_install_plugins:
   # pulp-ansible:
   #   source_dir: "/home/vagrant/devel/pulp_ansible"
+  # galaxy-pulp:
```
Comment (bmbouter) [inline example.dev-config.yml:7]: How'd you come up w/ this name?

Comment (fao89) [reply example.dev-config.yml:7]: it is the name of their bindings: https://github.com/ansible/galaxy_ng/blob/master/bindings/build.sh#L11

**Comment by fao89** [comment]
> Just to confirm, we do not need any other configs?

Last time I tried, I only needed this:
https://github.com/ansible/galaxy_ng/pull/1#issuecomment-597792052

**Review by bmbouter — APPROVED** [review]
Thank you!

**Pull request merged**